specs/experimental/admin-and-auth.md

```diff
@@ -25,21 +28,131 @@
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-The Security Council (at
-[eth:0xc2819DC788505Aac350142A7A707BF9D03E3Bd03](https://etherscan.io/address/0xc2819DC788505Aac350142A7A707BF9D03E3Bd03))
-uses a specially extended Safe multisig contract to provide additional security guarantees on top of
-those provided by the Safe contract.
+## Overview
+
+Authorization of administrative actions in the Superchain are managed by a system of multisignature
+ Safe contracts, as well as custom extensions to those Safe contracts which provide additional
+functionality. This document describes the system of Safe's and their purposes.
+
+## List of Safes
+
+This list outlines the various Safes, including mainnet addresses, thresholds, ownership, and any
+extensions.
+
+1. **[The ProxyAdminOwner
```

Comment:
We should document the safe versions we are using for each of these safes and note that this spec and the extensions are only compatible and tested with those versions

Reply:
ToDo: check my list of items to document. There may be something relevant to changes introduced in v1.4.0.